src/main.zig

```diff
@@ -340,11 +355,14 @@ fn completeGlobal(id: i64, pos_index: usize, handle: DocumentStore.Handle, confi
     // Deallocate all temporary data.
     defer arena.deinit();
 
+    var analysis_ctx = try document_store.analysisContext(handle, &arena, types.Position{.line = 0, .character = 0,});
```

Comment:
Can you remove the trailing comma in the position here?  
This will cause zig fmt to make it multiline

Reply:
zig fmt failed for some reason and I didn't notice, fixed now :)